.evergreen/install-mongodb-client-encryption.sh

```diff
@@ -0,0 +1,33 @@
+#! /usr/bin/env bash
+set +o xtrace # Do not write AWS credentials to stderr
+
+# Initiail checks for running these tests
+if [ -z ${INSTALL_DIR+omitted} ]; then echo "INSTALL_DIR is unset" && exit 1; fi
```

Comment:
```suggestion
# Initial checks for running these tests
```

Reply:
What change are you suggesting here?  Removing the guard on INSTALL_DIR?

Reply:
Oh this landed on the wrong line, did this review from vscode, `Initiail` -> `Initial`